docs/website/blog/2025-06-XX-client-cli-utxo-hd-ledger-state-snapshot-conversion

```diff
@@ -0,0 +1,28 @@
+---
+title: Mithril client CLI new 'tools utxo-hd snapshot-converter' command
+authors:
+  - name: Mithril Team
+tags: [mithril client, cli, command, utxo-hd, conversion]
+---
+
+### Introducing 'tools utxo-hd snapshot-converter' command
+
+With the release of the new distribution [2524](https://github.com/input-output-hk/mithril/releases/tag/2524.0), we have added the **tools** command in the **client CLI**.
+
+Since the release of the Cardano node [10.1.4](https://github.com/IntersectMBO/cardano-node/releases/tag/10.4.1), the Cardano database restoration downloads an `InMemory` UTxO-HD ledger state snapshot. However, if your Cardano node is configured for an on-disk format (`LMDB`) or the `legacy` UTxO-HD format, you need to convert that snapshot before running it.
```

Comment:
Can you:
- make more clear that the aggregator is creating snapshots with the `InMemory` flavor?
- explain that `Legacy` flavor is made for Cardano node 10.3 and previous versions?
- make clear that the conversion must be done before the restart of the Cardano node?